.github/workflows/jwa_frontend_tests.yaml

```diff
@@ -56,3 +56,65 @@ jobs:
         run: |
           cd components/crud-web-apps/jupyter/frontend
           npm run test:prod
+
+  run-tests-in-chrome:
+    name: UI tests in chrome
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup node version to 12
+        uses: actions/setup-node@v3
+        with:
+          node-version: 12
+
+      - name: Install Kubeflow common library dependecies
+        run: |
+          cd components/crud-web-apps/common/frontend/kubeflow-common-lib
+          npm i
+          npm run build
+          npm link ./dist/kubeflow
+      - name: Install JWA dependencies
+        run: |
+          cd components/crud-web-apps/jupyter/frontend
+          npm i
+          npm link kubeflow
+      - name: Serve UI & run Cypress tests in Chrome
+        uses: cypress-io/github-action@v4.2.0
+        with:
+          working-directory: components/crud-web-apps/jupyter/frontend
+          start: npm run serve
+          install: false
+          browser: chrome
+          wait-on: "http://localhost:4200"
+
+  run-tests-in-firefox:
+    name: UI tests in firefox
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup node version to 12
+        uses: actions/setup-node@v3
+        with:
+          node-version: 12
+
+      - name: Install Kubeflow common library dependecies
+        run: |
+          cd components/crud-web-apps/common/frontend/kubeflow-common-lib
+          npm i
+          npm run build
+          npm link ./dist/kubeflow
+      - name: Install JWA dependencies
+        run: |
+          cd components/crud-web-apps/jupyter/frontend
+          npm i
+          npm link kubeflow
+      - name: Serve UI & run Cypress tests in Firefox
+        uses: cypress-io/github-action@v4.2.0
+        with:
+          working-directory: components/crud-web-apps/jupyter/frontend
+          start: npm run serve
+          install: false
+          browser: firefox
+          wait-on: "http://localhost:4200"
```

Comment:
@orfeas-k since the setup for running the Firefox tests is exactly the same with the one for Chrome, what about moving this step into the one that runs the Chrome steps?

This way we avoid the duplicate work of running the tests twice

Reply:
The downside of merging the two actions would be that if in the future we don't mock the whole backend, then we'll need to break these actions again.

But I don't think we are going to move towards this direction for at least the next 2 months

Reply:
Sure, I have no strong preference over this so I 'm sending a fixup!

Reply:
Just pushed a fixup for this @kimwnasptd

Reply:
I think it's going to be a little bit involved to wait on localhost:4200 to be ready to serve `index.html`.

Let's see if we can get something to work. Else if it's too much trouble let's revert back